Jenkinsfile

```diff
@@ -1,4 +1,5 @@
 def workerNode = 'devel10'
```

Comment:
Skal vi over på devel11 i samme skynding?

Reply:
Nej for der kan vi ikke bygge gui. (npm er ikke cachet, og gui er ikke deployet til artifactory)